simple_history/tests/tests/test_utils.py

```diff
@@ -236,7 +236,9 @@ def test_bulk_create_with_history(self):
         self.assertEqual(BulkCreateManyToManyModel.objects.count(), 5)
 
 
-@skipIf(django.VERSION < (2, 2,), reason="bulk_update does not exist before 2.2")
+@skipIf(
+    django.VERSION < (2, 2,), reason="bulk_update does not exist before 2.2",
```

Comment:
Think we can remove these `skipIf` decorators?

Reply:
Yes, that seems right doesn't it. Let me have a look